sdk/core/Azure.Core/src/Pipeline/DiagnosticsOptions.cs

```diff
@@ -32,10 +38,9 @@ public DiagnosticsOptions()
         /// </summary>
         public int LoggingContentSizeLimit { get; set; } = 4 * 1024;
 
-        public IList<string> LoggingAllowedHeaderNames { get; }
-
-        public IList<string> LoggingAllowedQueryParameters { get; }
+        internal IList<string> LoggingAllowedHeaderNames { get; }
 
+        internal IList<string> LoggingAllowedQueryParameters { get; }
```

Comment:
So customers won't be able to change these?

Reply:
Not until I get a response from arch board. I really want this to be public. But if not at least they would have a convenient place to use reflection on :)